docs/v1.0/metrics/domains.md

```diff
@@ -158,17 +158,24 @@ Therefore, the P99 might actually be P98.9 or P99.2.
 Meta key `pN` indicates the configured percentile, and its value `pA` indicates the actual percentile that was used.
 
 * `truncate-table`<br>
-Default: no<br>
+Default: yes<br>
 Truncate [performance_schema.events_statements_histogram_global](https://dev.mysql.com/doc/refman/8.0/en/performance-schema-statement-histogram-summary-tables.html) after each collection.
-This reset percentile values so that each collection represents the global query response time during the collection interval rather than during the entire uptime of the MySQL.
+This resets percentile values so that each collection represents the global query response time during the collection interval rather than during the entire uptime of the MySQL.
 However, truncating the table interferes with other tools reading (or truncating) the table.
 
+* `truncate-timeout`<br>
+Default: 2s<br>
+The amount of time to wait while attempting to truncate [performance_schema.events_statements_histogram_global](https://dev.mysql.com/doc/refman/8.0/en/performance-schema-statement-histogram-summary-tables.html).
+
 #### Error Policy
 {: .no_toc }
 
 * `table-not-exist`<br>
 MySQL error 1146: Table 'performance_schema.events_statements_histogram_global' doesn't exist
 
+* `truncate-failed`<br>
```

Comment:
s//truncate-timeout/ so it matches the option, creating a nice symmetry: here's the timeout option and here's what to do if that timeout errors.

Reply:
Can do. To be clear, I'm triggering this on _any_ failure with the truncate, not just timeouts. A timeout is those most likely event, but really we want to avoid emitting metrics if we have truncation failures in general.